tests/run_examples.py

```diff
@@ -157,9 +157,9 @@ def check_library(rule):
 
 req_library = defaultdict(lambda: [])
 req_library['basic_statistics_spmd.py'] = ['dpctl', 'mpi4py']
-req_library['gbt_cls_model_create_from_lightgbm_batch.py'] = ['lightgbm']
-req_library['gbt_cls_model_create_from_xgboost_batch.py'] = ['xgboost']
-req_library['gbt_cls_model_create_from_catboost_batch.py'] = ['catboost']
+req_library['model_builders_lightgbm.py'] = ['lightgbm']
+req_library['model_builders_xgboost.py'] = ['xgboost']
+req_library['model_builders_catboost.py'] = ['catboost']
```

Comment:
No. Batch is strange and miningless suffix. If runner will not pick up them, then launcher should be fixed

Reply:
Let's decide how we can group-by this kind of examples then.
All examples [scikit-learn-intelex/examples/](https://github.com/intel/scikit-learn-intelex/tree/master/examples/) are all grouped logically into 4 groups: `spmd.py`, `streaming.py`, `stream.py`, `batch.py`.
If none of them is suitable, then:
* we will come up with a new suffix for such examples.
* update whole examples runner logic

Reply:
I have just made a brief look in the run_examples.py. May we just add some sort of default behavior in case the filename doesn't end on `*spmd.py`, `*streaming.py`, `*stream.py` or `*batch.py`?
As far as I see we could switch to this default behavior for all the `*_batch.py` and `*_stream.py` files. In my mind, it will make naming of examples more clear.

Reply:
@razdoburdin i like that idea more, than using suffixes.  

@samir-nasibli - those are logical only if you look on this from some testing perspective. From end user perspective this looks strange.

I would go with base examples without batch because it make no sense to specify this. On stream/streaming - looks like this should have be single group. And for spmd - let's leave it as is

@Alexsandruss, @KulikovNikita   - your thoughts?

Reply:
> I have just made a brief look in the run_examples.py. May we just add some sort of default behavior in case the filename doesn't end on `*spmd.py`, `*streaming.py`, `*stream.py` or `*batch.py`?

Sure we can. That means that we need update examples runner logic little bit (second option).